### PR TITLE
Make access to logging and monitoring more self-explanatory from the …

### PR DESCRIPTION
**Pull request by BeckerMax**
…docs

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```improvement operator

```

### CODE REVIEW

File: website/documentation/getting-started/observability/components.md [hunk 1]
```diff
@@ -36,13 +40,24 @@ The password you can use to log in can be retrieved as shown in the image below.
 
 ![](./images/access-plutono.png)
 
-After logging into Plutono, you will see dashboards like the one below. The depicted dashboard, for example, offers an overview of the Kubernetes Control Plane availability.
+### Accessing the dashboards
+
+After logging in you will be greeted with a Plutono welcome screen. Click on `General/Home` at the top as depicted with the red arrow in the next picture:
 
+![](./images/welcome_plutono.png)
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:47]: I can't seem to find this image in the PR, or any of the other newly added images.

File: website/documentation/getting-started/observability/components.md [hunk 2]
```diff
@@ -36,13 +40,24 @@ The password you can use to log in can be retrieved as shown in the image below.
 
 ![](./images/access-plutono.png)
 
-After logging into Plutono, you will see dashboards like the one below. The depicted dashboard, for example, offers an overview of the Kubernetes Control Plane availability.
+### Accessing the dashboards
+
+After logging in you will be greeted with a Plutono welcome screen. Click on `General/Home` at the top as depicted with the red arrow in the next picture:
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:45]:
```suggestion
After logging in, you will be greeted with a Plutono welcome screen. Navigate to `General/Home`, as depicted with the red arrow in the next picture:
```

File: website/documentation/getting-started/observability/components.md [hunk 3]
```diff
@@ -36,13 +40,24 @@ The password you can use to log in can be retrieved as shown in the image below.
...
 
+![](./images/welcome_plutono.png)
+
+Then you will be able to select the dashboards. Some interesting ones to look at are:
+- `Kubernetes Control Plane Status` if you want to check control plane availability during a certain time frame go to the this dashboard.
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:50]:
```suggestion
- The `Kubernetes Control Plane Status` dashboard allows you to check control plane availability during a certain time frame.
```

File: website/documentation/getting-started/observability/components.md [hunk 4]
```diff
@@ -36,13 +40,24 @@ The password you can use to log in can be retrieved as shown in the image below.
...
+- The `API Server` dashboard gives you an overview which requests are done towards your apiserver and how long they take.
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:51]:
```suggestion
- The `API Server` dashboard gives you an overview on which requests are done towards your apiserver and how long they take.
```

File: website/documentation/getting-started/observability/components.md [hunk 5]
```diff
@@ -56,10 +71,30 @@ Time series data from Prometheus can be made visible with Plutono. Here we see h
 
 ### Vali Logs via Plutono
 
-Vali is our logging solution. Here we see how logs gathered with Vali are visualized in Plutono. Therefore, we can correlate logs and metrics in the same UI.
+Vali is our logging solution. In order to access the logs provided by Vali first log into Plutono as describe [here](https://gardener.cloud/docs/getting-started/observability/components/#logging-into-Plutono).
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:74]:
```suggestion
Vali is our logging solution. In order to access the logs provided by Vali, you need to:

1. Log into Plutono, as described [here](https://gardener.cloud/docs/getting-started/observability/components/#logging-into-Plutono).
```

File: website/documentation/getting-started/observability/components.md [hunk 6]
```diff
@@ -56,10 +71,30 @@ Time series data from Prometheus can be made visible with Plutono. Here we see h
...
+
+Then click on Explore which is depicted as the little compass symbol. See the red arrow in the next picture for where to click:
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:76]:
```suggestion
2. Choose `Explore`, which is depicted as the little compass symbol:
```

File: website/documentation/getting-started/observability/components.md [hunk 7]
```diff
@@ -56,10 +71,30 @@ Time series data from Prometheus can be made visible with Plutono. Here we see h
...
+
+ ![](images/explore_loki.png)
+
+Then select `Vali` at the top left as shown here:
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:80]:
```suggestion
3. Select `Vali` at the top left, as shown here:
```

File: website/documentation/getting-started/observability/components.md [hunk 8]
```diff
@@ -56,10 +71,30 @@ Time series data from Prometheus can be made visible with Plutono. Here we see h
...
+
+![](./images/select_vali.png)
+
+There you can browse logs or events of the control plane components.
 
 ![](./images/vali-logs.png)
 
+Helpful queries would be for example:
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:88]:
```suggestion
Here are some examples of helpful queries:
```

File: website/documentation/getting-started/observability/components.md [hunk 9]
```diff
@@ -56,10 +71,30 @@ Time series data from Prometheus can be made visible with Plutono. Here we see h
...
+- `{container_name="cluster-autoscaler" }` to get cluster-autoscaler logs and see why certain node groups were scaled up.
+- `{container_name="kube-apiserver"} |~ "error" ` to get the logs of the kube-apiserver container and filter for errors.
+- `{unit="kubelet.service", nodename="ip-123"}` gives the kubelet logs of a specific node
+- `{unit="containerd.service", nodename="ip-123"}` retrieves the containerd logs for a specific node
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:92]:
```suggestion
- `{unit="kubelet.service", nodename="ip-123"}` to get the kubelet logs of a specific node.
- `{unit="containerd.service", nodename="ip-123"}` to retrieve the containerd logs for a specific node.
```

File: website/documentation/getting-started/observability/components.md [hunk 10]
```diff
@@ -56,10 +71,30 @@ Time series data from Prometheus can be made visible with Plutono. Here we see h
...
+
+ See also the `Help >` button to see options exist to filter the results.
+
+ For more information also how to retrieve K8s events from the past see [How to access Logs](https://github.com/gardener/gardener/blob/master/docs/usage/logging.md#how-to-access-the-logs)
```
Comment (n-boshnakov) [inline website/documentation/getting-started/observability/components.md:96]:
```suggestion
 Choose `Help >` in order to see what options exist to filter the results.

 For more information on how to retrieve K8s events from the past, see [How to Access Logs](https://github.com/gardener/gardener/blob/master/docs/usage/logging.md#how-to-access-the-logs).
```

**Review by n-boshnakov — CHANGES_REQUESTED** [review]

**Review by n-boshnakov — APPROVED** [review]
/lgtm

**Review by JordanJordanov — APPROVED** [review]
/lgtm

**Pull request merged**